scipy/stats/tests/test_distributions.py

```diff
@@ -2715,6 +2726,41 @@ def test_edge_cases(self):
             assert_equal(stats.triang.cdf(1., 1.), 1)
 
 
+class TestBurr(object):
+    def test_endpoints_7491(self):
+        # gh-7491
+        # Compute the pdf at the left endpoint dst.a.
+        data = [
```

Comment:
Should there be a unit test here for the `f` distribution as well? 
It might make sense for a separate class, or if kept here change the test class name to something like `TestParameterDependentEndpoints(object):` with different tests method for the `burr` family of distributions and one for `f` endpoints (2 separate methods in the test class. Given the pecularities of the distributions this is not likely to be impacting each distribution and requires some specific domain expertise, so I'd prefer a single test class with 2 methods but I'm not firm on that if there are good arguments to push these tests out to each distribution.

Reply:
The `f` has its own test of the endpoints `TestF::test_endpoints`.